Ops/Ops-lessons-2022/AWS.md

```diff
@@ -71,16 +71,67 @@ Documentation:
 ### Cloudwatch
 
 ### EventBridge
+- Abstracted version of cloudwatch events
+- Allows users to create rules that initiate events that trigger a target it has set in that rule
+- Eventbus is a collection of rules that triggers targets
+- In eventbus an event has an rule and a target
```

Comment:
An event doesn't have a rule but eventbus is more of like a queue where if an event put to the event bus matches a rule then it will try and invoke the target